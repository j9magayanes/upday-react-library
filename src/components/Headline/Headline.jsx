import React from 'react';
import './headline.css';


export const Headline= ({args}) => {
        return (
    
            <div className="padded-lr legal">
                <h1 id="imprint">{args.headline}</h1>
            </div>
    )}
