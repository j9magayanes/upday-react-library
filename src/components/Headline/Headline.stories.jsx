import React from 'react';

import { Headline } from './Headline';


export default {
  title: 'Form/Headline',
  component: Headline,
};

const Template = () =><Headline args={{headline: "Headline"}}  />;


export const Default = Template.bind({});