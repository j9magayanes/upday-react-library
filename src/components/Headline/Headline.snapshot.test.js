import React from 'react';
import renderer from 'react-test-renderer';
import { Headline } from './Headline';

test('renders Headline component correctly', () => {
  const props = {
    args: {
      headline: 'Your headline text here',
      // Add other required properties as needed
    },
  };

  const component = renderer.create(<Headline {...props} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
