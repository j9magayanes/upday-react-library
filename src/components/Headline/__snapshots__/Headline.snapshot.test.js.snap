// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Headline component correctly 1`] = `
<div
  className="padded-lr legal"
>
  <h1
    id="imprint"
  >
    Your headline text here
  </h1>
</div>
`;
